models/migrations/v85.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"github.com/go-xorm/xorm"
+)
+
+func addTeamIsAllRepositories(x *xorm.Engine) error {
+
+	type Team struct {
+		ID                int64 `xorm:"pk autoincr"`
+		IsAllRepositories bool  `xorm:"NOT NULL DEFAULT false"`
+	}
+
+	if err := x.Sync2(new(Team)); err != nil {
+		return err
+	}
+
+	_, err := x.Exec("UPDATE team SET is_all_repositories = ? WHERE name=?",
```

Comment:
You should set backticks around table/column names

Reply:
Will do